--- Running scipy.integrate.quad_vec#56YY/quad_vecYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.integrate.quad_vec#56YY'), ('notebook', 'quad_vecYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.523s
user	0m2.239s
sys	0m8.219s
